pkg/cli/cmd/env/update/update_test.go

```diff
@@ -30,7 +30,6 @@ import (
 	"github.com/radius-project/radius/pkg/cli/framework"
 	"github.com/radius-project/radius/pkg/cli/output"
 	"github.com/radius-project/radius/pkg/cli/workspaces"
-	"github.com/radius-project/radius/pkg/corerp/api/v20231001preview"
```

Comment:
Is this something we could turn on in `golanglint-ci`? I think this is a good change, I'm just wondering how we could automate it.

Reply:
I found it by using `staticcheck ./...`. We can add that to a CI/CD we have.

Reply:
I did some research, it sounds like we should add staticcheck. I thought it was bundled into golanglint-ci but it isn't exactly, sounds like there is a history there.

Reply:
Created this feature request: https://github.com/radius-project/radius/issues/7795. cc/ @rynowak